### ISSUE TITLE
Improve Windows install instructions

### ISSUE THREAD

**Issue by silbinarywolf** [pions/webrtc#203]
Hi there!

I'm currently interested in testing out / trying this project so I can experiment with getting a UDP-like solution in the browser. However I have no clue how to get openssl built on Windows and I'm getting stuck on step 3. I'm just not able to find an OpenSSL package that gives me any sort of ".pc" file.

I've tried installing this but it doesn't seem to come with the correct file:
- https://slproweb.com/products/Win32OpenSSL.html / Win64OpenSSL-1_1_1.msi

Can the documentation be updated to link to a working precompiled OpenSSL download?

**Current error messages:**
```
C:\Users\Jake>go get -u github.com/pions/webrtc
# pkg-config --cflags  -- libssl libcrypto
Package libssl was not found in the pkg-config search path.
Perhaps you should add the directory containing `libssl.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libssl' found
Package libcrypto was not found in the pkg-config search path.
Perhaps you should add the directory containing `libcrypto.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libcrypto' found
pkg-config: exit status 1
```

**Current instructions in README.md**
#### Windows
1. Install [mingw-w64](http://mingw-w64.sourceforge.net/)
2. Install [pkg-config-lite](http://sourceforge.net/projects/pkgconfiglite)
3. Build (or install precompiled) openssl for mingw32-w64
4. Set __PKG\_CONFIG\_PATH__ to the directory containing openssl.pc
   (i.e. c:\mingw64\mingw64\lib\pkgconfig)

**Comment by backkem** [comment]
Hi @silbinarywolf , thanks for the feedback.
Here's what I did to install the dependencies on Windows:
1. Install [MSYS2](https://www.msys2.org/)
2. Download [pkg-config-lite](https://sourceforge.net/projects/pkgconfiglite/) and put `pkg-config.exe` on the `PATH` environment variable. I did this by placing the `pkg-config.exe` executable in `%GOPATH%/Bin` since this directory is part of my `PATH` environment variable.
3. From the MSYS2 console: Run `pacman -S mingw-w64-x86_64-openssl` to install OpenSSL.
4. From the MSYS2 console: Run `pacman -S mingw-w64-x86_64-gcc` to install the C compiler.
5. Set the `PKG_CONFIG_PATH` environment variable. I did this by running the following from the Windows console: Run `SETX PKG_CONFIG_PATH C:\msys64\mingw64\lib\pkgconfig`

Let me know if you have any more questions.
